airflow/utils/log/json_formatter.py

```diff
@@ -43,5 +43,16 @@ def usesTime(self):
     def format(self, record):
         super().format(record)
         record_dict = {label: getattr(record, label, None) for label in self.json_fields}
+        if "message" in self.json_fields:
+            msg = record_dict["message"]
+            if record.exc_text:
+                if msg[-1:] != "\n":
+                    msg = msg + "\n"
+                msg = msg + record.exc_text
+            if record.stack_info:
+                if msg[-1:] != "\n":
+                    msg = msg + "\n"
+                msg = msg + self.formatStack(record.stack_info)
```

Comment:
This is from https://github.com/python/cpython/blob/adf24bd835ed8f76dcc51aa98c8c54275e86965b/Lib/logging/__init__.py#L687-L694